packages/flex-plugin-webpack/src/devServer/pluginServer.ts

```diff
@@ -167,6 +204,7 @@ export const _startServer = (
 
     const hasRemotePlugin = config.remoteAll || plugins.remote.length !== 0;
     const localPlugins = _getLocalPlugins(config.port, plugins.local);
+    const versionedPlugins: Plugin[] = plugins.versioned ? _getRemoteVersionedPlugins(plugins.versioned) : [];
```

Comment:
`plugins.versioned` should always be defined and be empty if nothing is in it - i.e you should be able to do `const versionedPlugins = _getRemoteVersionedPlugins(plugins.versioned)`